### DISCUSSION TITLE
Vidstack/Poster with vertical videos

### DISCUSSION THREAD

**Discussion by dennisspohr** [vidstack/player#1006] [Q&A]
Hi Community!

One of our clients uses vertical videos. The video itself is displayed properly, but the poster before starting the video is not shown correctly. The poster covers the whole area and gets cropped. I should be contained, which a black area on the left/right of the poster. Same like the video. I tried to fix the poster via CSS - but no luck.

Do you have any idea?
Thanks!

**Comment by wplit** [comment]
The default behaviour, at least with the default layout CSS, seems to be already you're wanting it, contained and not cropped.

https://codepen.io/wplit/pen/mdaNyjz

the img has   'object-fit: contain' by default. You may need to share an example to see why it's not behaving this way in your case.

**Comment by dennisspohr** [comment]
Here's an example (we are using Vidstack 0.6.13, because we had randomly some loading-problems in the latest version):

https://webschool.coachy.net/video-test/hochkant-video-poster/hochkant/

**Reply by mihar-22** [reply]
There's been tons of bug fixes since we launched the 1.0 pre-release. You'll most likely find your issues resolved in latest `1.5.6@next`. Please give it a shot and let us know if there's any issues (open GitHub issue). I can get loading issues patched and released straight away.

**Reply by dennisspohr** [reply]
@mihar-22 alright, I will test it again. We have quite a lot of users using our platform, so we always have to be careful. With the loading-problems last time it was hard to find a system behind it, it happened quite randomly.

**Reply by dennisspohr** [reply]
@mihar-22 Using the latest version on our dev-server now. Can't put it live yet, because there seems to be another issue. 
Initially it looks good now. However we have the feature that the player saved the currently played time in the local storage. If someone reloads the page, I set the `player.currentTime = seconds;`. But now it has the effect, that both the video and the poster are shown. Since the player doesn't start playing, it should only show the poster at that stage.

**Reply by mihar-22** [reply]
Thanks for upgrading and trying it out! I didn't fully understand the issue. You're saying the poster is hidden when you set the current time but you want it to show? Technically playback has started if you seek to a new time (i.e., set current time), so not sure we'd consider it a bug. If you can open a GitHub issue with some info then we can talk about it, it'll also help us track the issue since it's not exactly related to this discussion.
